FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/FhirToDataLakeOrchestratorJob.cs

```diff
@@ -88,13 +91,20 @@ public class FhirToDataLakeOrchestratorJob : IJob
 
         public int NumberOfPatientsPerProcessingJob { get; set; } = JobConfigurationConstants.DefaultNumberOfPatientsPerProcessingJob;
 
+        public int LowBoundOfProcessingJobResourceCount { get; set; } = JobConfigurationConstants.LowBoundOfProcessingJobResourceCount;
+
+        public int HighBoundOfProcessingJobResourceCount { get; set; } = JobConfigurationConstants.HighBoundOfProcessingJobResourceCount;
+
         public async Task<string> ExecuteAsync(IProgress<string> progress, CancellationToken cancellationToken)
         {
             _diagnosticLogger.LogInformation("Start executing FhirToDataLake job.");
             _logger.LogInformation($"Start executing FhirToDataLake orchestrator job {_jobInfo.Id}.");
 
             try
             {
+                _jobStatus = await InitializeJobStatusAsync(cancellationToken);
```

Comment:
how about assgining and reporting result in the JobStatus initialize and update functions?

Reply:
Maybe update status and report results are two operations. Still separate them but only serialize result string one time.